### PR TITLE
Adding option to use pre-existing secrets

### PR DESCRIPTION
**Pull request by tjespers**
Signed-off-by: Tim Jespers <t.jesp3rs@gmail.com>

#### What this PR does / why we need it:
This PR allows the helm operator to use a pre-existing kubernetes secret instead of supplying their base64 service account keyfile as a release value. Doing so allows the operator to keep the service account keyfile completely out of helm and/or source-control when storing their value file in a git repository.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://developercertificate.org) signed
- [x] Chart Version bumped (if the pr is an update to an existing chart)
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by tjespers** [comment]
@rpahli

File: charts/gcp-serviceaccount-controller/values.yaml [hunk 1]
```diff
@@ -7,6 +7,12 @@ nameOverride: ""
 fullnameOverride: ""
 
 gcpCredentials: ""
+
+# Use a pre-existing secret (ignoring gcpCredentials)
+existingSecret: ""
```
[the remark this answers is not in the document]
Comment (tjespers) [reply charts/gcp-serviceaccount-controller/values.yaml]: @rpahli sure thing! done

**Comment by tmc** [comment]
The notes file should be updated.

**Comment by tjespers** [comment]
@rpahli @tmc i've updated the notes as well to reflect the changes. So we should be good to go now i think?

File: charts/gcp-serviceaccount-controller/values.yaml [hunk 2]
```diff
@@ -7,6 +7,12 @@ nameOverride: ""
 fullnameOverride: ""
 
 gcpCredentials: ""
+
+# Use a pre-existing secret (ignoring gcpCredentials)
+existingSecret: null
```
[the remark this answers is not in the document]
Comment (tjespers) [reply charts/gcp-serviceaccount-controller/values.yaml]: @rpahli changed, as by default the value adds nothing if not set i also took the liberty to comment the value so one could enable it using a single `--set` argument

e.g. 
`helm install kiwigrid/gcp-serviceaccount-controller --set existingSecret=my-secret`

**Comment by tjespers** [comment]
@rpahli do we need something else to get this merged? :)

**Pull request merged**